src/pages/index.js

```diff
@@ -176,14 +295,41 @@ function openProfileForm() {
 
 // Event Listeners
 profileEditButton.addEventListener("click", openProfileForm);
+
 profileModalCloseButton.addEventListener("click", () =>
```

Comment:
Please, delete close icon listeners from `index.js`. They are handled in `Popup`

Reply:
completed.